tst/teststandard/opers/StructureDescription.tst

```diff
@@ -2,4 +2,8 @@ gap> START_TEST("StructureDescription.tst");
 gap> G := Group([ (6,7,8,9,10), (8,9,10), (1,2)(6,7), (1,2,3,4,5)(6,7,8,9,10) ]);;
 gap> StructureDescription(G);
 "A5 : S5"
+
+## this currently takes a veeeeeery long time (about 20 minutes)
+#gap> StructureDescription(Ree(27));
+#"Ree(27)"
```

Comment:
I made it work with a hack, see above ````tst/testinstall/opers/StructureDescription.tst:87````. This would be cleaner, but I agree with what @hulpke says in #987, that it is not a real usecase. I could just remove this if you like (but it is commented out anyway).

Just to note that this example was not entirely in vain: the particular line this runs in ````StructureDescription```` _was_ _incorrect_, and this test made me correct it in this PR.